### PR TITLE
lastRatingChangeTimestamp

### PR DESCRIPTION
**Pull request by Tadly**
added some info for lastRatingChangeTimestamp

### CODE REVIEW

File: gmusicapi/protocol/mobileclient.py [hunk 1]
```diff
@@ -73,6 +73,8 @@
         'genre': {'type': 'string', 'required': False},
         'trackAvailableForSubscription': {'type': 'boolean'},
         'contentType': {'type': 'string'},
+        # Only availabel when rating differs from '0'
```
Comment (simon-weber) [inline gmusicapi/protocol/mobileclient.py]: got a typo here.

File: gmusicapi/protocol/mobileclient.py [hunk 2]
```diff
@@ -73,6 +73,8 @@
...
+        # when using :change_song_metadata:, specifying this value will cause all clients to properly update (web/mobile). As value int(round(time.time() * 1000000)) works quite well
```
Comment (simon-weber) [inline gmusicapi/protocol/mobileclient.py]: let's split this long line; that's what caused linting to fail.

**Comment by simon-weber** [comment]
Thanks! Just two quick fixes before we merge.

**Comment by Tadly** [comment]
btw. all fixed now (I hope) ;)

**Comment by simon-weber** [comment]
Looks good; thanks!

**Pull request merged**